spring/boot2-actuator-autoconfigure/src/main/java/com/linecorp/armeria/spring/actuate/WebOperationService.java

```diff
@@ -127,7 +128,11 @@ public HttpResponse serve(ServiceRequestContext ctx, HttpRequest req) {
                 return null;
             }
             if (operation.isBlocking()) {
-                ctx.blockingTaskExecutor().execute(() -> invoke(ctx, aggregatedReq, resFuture));
+                try {
+                    ctx.blockingTaskExecutor().execute(() -> invoke(ctx, aggregatedReq, resFuture));
+                } catch (RejectedExecutionException ree) {
+                    resFuture.completeExceptionally(ree);
+                }
```

Comment:
How about just catching a `Throwable`?

```java
req.aggregate().handle(... -> {
  if (t != null) { ... }
  try {
    ..
  } catch (Throwable cause) {
    resFuture.completeExceptionally(cause);
  }
  return null;
})
```

Reply:
Oh.. how does that make thing better?

Reply:
- It will catch other exceptions than `RejectedExecutionException`
- We're absolutely sure nothing will ever fail like before, even when `invoke()` below throws an exception.

Reply:
I suppose you mean that custom `ExecutorService` who throw other runtime exceptions or even errors can't cause trouble here?
Okay then